### PR TITLE
Overwritable routes file

### PR DESCRIPTION
**Pull request by tabacitu**
Allows us to overwrite the package routes by creating the ```routes/backpack/logmanager.php``` file.

### CODE REVIEW

File: src/LogManagerServiceProvider.php [hunk 1]
```diff
@@ -16,6 +16,13 @@ class LogManagerServiceProvider extends ServiceProvider
     protected $defer = false;
 
     /**
+     * Where the route file lives, both inside the package and in the app (if overwritten).
+     *
+     * @var bool
```
Comment (lloy0076) [inline src/LogManagerServiceProvider.php]: Is that a bool?

File: src/routes/backpack/logmanager.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+|
+*/
+
+Route::group([
+            'namespace'  => 'Backpack\LogManager\app\Http\Controllers',
+            'middleware' => ['web', 'admin'],
+            'prefix'     => config('backpack.base.route_prefix', 'admin'),
+    ], function () {
+
+    // LogManager
```
Comment (lloy0076) [inline src/routes/backpack/logmanager.php]: Is this comment necessary?

File: src/routes/backpack/logmanager.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Backpack\LogManager Routes
+|--------------------------------------------------------------------------
+|
+| This file is where you may define or change all of the routes that are
```
Comment (lloy0076) [inline src/routes/backpack/logmanager.php]: Technically we're not really changing anything here - it's just defining everything.

File: src/routes/backpack/logmanager.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+| handled by the Backpack\LogManager package.
+|
+*/
+
+Route::group([
+            'namespace'  => 'Backpack\LogManager\app\Http\Controllers',
```
Comment (lloy0076) [inline src/routes/backpack/logmanager.php:14]: Ok, sometimes these are aligned and sometimes they are not; I prefer when they are but I think StyleCI is set to not align => and = (or maybe it's just personal preference).

File: src/routes/backpack/logmanager.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+            'middleware' => ['web', 'admin'],
+            'prefix'     => config('backpack.base.route_prefix', 'admin'),
+    ], function () {
+
+    // LogManager
+    Route::get('log', 'LogController@index');
+        Route::get('log/preview/{file_name}', 'LogController@preview');
```
Comment (lloy0076) [inline src/routes/backpack/logmanager.php]: Why are they indented?

Comment (tabacitu) [inline src/routes/backpack/logmanager.php]: It is not, you're right.

File: src/routes/backpack/logmanager.php (hunk 2 above)
Comment (tabacitu) [reply src/routes/backpack/logmanager.php]: It is not, you're right.

File: src/routes/backpack/logmanager.php (hunk 5 above)
Comment (tabacitu) [reply src/routes/backpack/logmanager.php]: StyleCI, I think...

File: src/routes/backpack/logmanager.php (hunk 4 above)
Comment (tabacitu) [reply src/routes/backpack/logmanager.php:14]: I think so, yes. Would change this, but I'm sure StyleCI would change it back to how it likes it :-)

**Pull request merged**